### ISSUE TITLE
Process out of memory extracting material design icons

### ISSUE THREAD

**Issue by guybedford** [bower/decompress-zip#48]
If I download the zip file from the release at https://github.com/google/material-design-icons/releases/tag/2.0.0, and run that directly through the standard extraction from this readme, I get the following error in Node:

```
FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - process out of memory
Abort trap: 6
```

Any ideas on this one welcome!

**Comment by samccone** [comment]
hey @guybedford thanks for the post I xposted on the google repo for the time being

**Comment by guybedford** [comment]
Thanks for the quick response, I don't think the bower version downloads the release zip file as far as I can tell? Otherwise I'd have thought this would be flagged already.

**Comment by samccone** [comment]
ok, cool so the size of the zip is clearly the issue here

**Comment by samccone** [comment]
@guybedford can you give this a try https://github.com/thejoshwolfe/yauzl

**Comment by guybedford** [comment]
@samccone yeah that works perfectly actually. I did also try raising the memory limit here but it still ends up going over 1GB which seems odd for a 50MB file.
